<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonGroup</class>
 <widget class="QWidget" name="ButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>67</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>游客</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_user.png</normaloff>:/images/main/title_user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnTransform">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>10</y>
     <width>107</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>传输列表</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_data.png</normaloff>:/images/main/title_data.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDownload">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>10</y>
     <width>87</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下载榜</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_hot.png</normaloff>:/images/main/title_hot.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSwitchUser">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>10</y>
     <width>107</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>切换用户</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_data.png</normaloff>:/images/main/title_data.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnMyFile">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>10</y>
     <width>107</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我的文件</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_file.png</normaloff>:/images/main/title_file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btnShare">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>107</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>共享列表</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main/title_share.png</normaloff>:/images/main/title_share.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>137</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnMin">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/main/window/title_min.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnMax">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/main/window/title_max.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnClose">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/main/window/title_close.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
